lang/pt_BR.json

```diff
@@ -228,19 +228,19 @@
     "Comments on my user wall": "Comentários na minha parede de usuário",
     "Comments on a forum topic I'm involved in": "Comentários em um tópico do fórum do qual participo",
     "Someone follows me": "Alguém me segue",
-    "I receive a private message": "Recebo uma mensagem privada",
+    "I receive a private message": "Recebi uma mensagem privada",
     "Ticket activity": "Atividade de Ticket",
     "Email me": "Me envie um e-mail",
     "Notify me on the site": "Notificar-me no site",
     "Preferences": "Preferências",
-    "Suppress mature content warnings": "Suprimir avisos de conteúdo maduro",
-    "Show absolute dates on forum posts": "Mostrar datas absolutas nas postagens do fórum",
+    "Suppress mature content warnings": "Ocultar avisos de conteúdo impróprio",
+    "Show absolute dates on forum posts": "Exibir datas completas nas postagens do fórum",
     "Hide missable achievement indicators": "Ocultar indicadores de conquistas perdidas",
     "Only people I follow can message me or post on my wall": "Apenas pessoas que sigo podem me enviar mensagens ou postar na minha parede",
     "Automatically opt in to all game sets": "Inscrever-se automaticamente em todos os conjuntos de jogos",
-    "Site Awards": "Prêmios do Site",
-    "You can manually reorder how badges appear on your user profile.": "Você pode reordenar manualmente como os emblemas aparecem no seu perfil de usuário.",
-    "Reorder Site Awards": "Reordenar Prêmios do Site",
+    "Site Awards": "Platinas da conta",
```

Comment:
This is the only change I feel that probably is worth more discussion.

There are multiple kinds of site awards that aren't just masteries. For example, there are awards for reaching certain achievement developer milestones. Being a Patron gets an award. Etc.

Reply:
I agree with you, but the word 'Award' goes beyond a mastered or a badge. For us, the word 'Award' means that you won some competition, whether it’s a race, soccer, or any other sport. The word loses its meaning; what could change would be the badges, and you could call them 'Emblems,' or keep them as 'Platinas e Emblemas' as well, since in Portuguese they are different words (quite different).